modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -0,0 +1,55 @@
+= Upgrade to Anypoint Service Mesh 1.1
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, upgrading
+
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as the authorization mechanism aspect of the mixer is now implemented by leveraging Anypoint Service Mesh Envoy filters. Note that the architectural changes in the upgrade require some downtime.
+
+When you upgrade from Anypoint Service Mesh v1.1, the process first creates a backup of the necessary files. Any request to protected services is denied and the Anypoint Service Mesh downtime begins. 
+
+The adapters, APIs, and bindings are then upgraded, and finally requests to protected services are resumed, thereby ending the Anypoint Service Mesh downtime. 
+
+
+== Prerequisites
+
+Before you upgrade, ensure that your environment uses:
+
+* Anypoint Service Mesh v1.0.x
+* The most recent version of the Anypoint Service Mesh command line interface (CLI)
++
+The CLI version must be v1.1.x and the Anypoint Service Mesh version must be v1.0.x.
+* Istio 1.6.0 or later for both control plane and data plane
```

Comment:
Ask Kim if we have formal terms for these, yet? We should not use data plane. Use runtime plane instead.